src/main/java/de/retest/recheck/ioerror/ReTestIOException.java

```diff
@@ -2,24 +2,27 @@
 
 import java.io.File;
 import java.io.IOException;
-
-import de.retest.recheck.util.FileUtil;
+import java.net.URI;
 
 public class ReTestIOException extends IOException {
 
 	private static final long serialVersionUID = 1L;
-	private final File file;
+	private final URI location;
 
 	public ReTestIOException( final File file, final Throwable throwable ) {
```

Comment:
Since this is an internal exception and we now use `URI` instead of `File` (see e.g. `FileUtil`), I would remove this ctor to reduce the amount of code we have to maintain. Same goes for `ReTestLoadException` and `ReTestSaveException`.

Reply:
OK